<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskTestParamsEditor</class>
 <widget class="QDialog" name="TaskTestParamsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Edit Task</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="comparatorLabel">
       <property name="text">
        <string>Compare Type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comparatorComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="cpuLabel">
       <property name="text">
        <string>Cpu:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="cpuBox">
       <property name="suffix">
        <string notr="true"> milliseconds</string>
       </property>
       <property name="maximum">
        <number>1000000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="memLabel">
       <property name="text">
        <string>Mem:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="memBox">
       <property name="suffix">
        <string notr="true"> megabytes</string>
       </property>
       <property name="maximum">
        <number>1025</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="qryLabel">
       <property name="text">
        <string>Qry:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="qryBox">
         <property name="suffix">
          <string notr="true"/>
         </property>
         <property name="maximum">
          <number>101</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="interactiveCheckBox">
         <property name="text">
          <string>Interactive ?</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
